pkg/api/v1/register.go

```diff
@@ -86,6 +86,10 @@ func addKnownTypes() {
 		&ThirdPartyResourceList{},
 		&ThirdPartyResourceData{},
 	)
+
+	// Add common types
+	api.Scheme.AddKnownTypes("v1", &unversioned.Status{})
```

Comment:
I'm not sure if this is the right way. Depends on whether "v1" or "" is registered first, type unversioned.Status{} will be marked as "v1" or "" in the schema. I don't know if this is going to cause any problem though.

It 's safer if we just keep the v1.Status{} and register it here.